<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">AdminCentre</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="mainStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>700</height>
     </rect>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>491</width>
        <height>85</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="eventsPage">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>1080</width>
        <height>580</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1078</width>
         <height>578</height>
        </rect>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1061</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="eventsLayout"/>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="addEventButton">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>10</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveAllButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>10</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Save All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reloadDataButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Reload Data</string>
      </property>
     </widget>
     <widget class="QComboBox" name="langBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string notr="true">English</string>
       </property>
       <property name="icon">
        <iconset resource="../res/qrc/flags.qrc">
         <normaloff>:/lang_icons/uk.png</normaloff>:/lang_icons/uk.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">Russian</string>
       </property>
       <property name="icon">
        <iconset resource="../res/qrc/flags.qrc">
         <normaloff>:/lang_icons/ru.png</normaloff>:/lang_icons/ru.png</iconset>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../res/qrc/flags.qrc"/>
 </resources>
 <connections/>
</ui>
